docs/editor_guide/introduction.rst

```diff
@@ -0,0 +1,23 @@
+Introduction
+============
+
+Wagxperience_ is a brand spanking new open source module for the Wagtail_
+content management system developed by `Lab Digital`_. It allows editors and
```

Comment:
misschien nog beter om te beginnen met: Wagexperience is developed by lab digital and is a open-source module for Wagtail..

_-- Het blijft niet brand new, op deze manier hoeven we dat straks niet aan te passen_